src/server/pfs/server/testing/server_v2_test.go

```diff
@@ -694,14 +690,10 @@ func finfosToPaths(finfos []*pfs.FileInfo) (paths []string) {
 }
 
 func TestListFileV2(t *testing.T) {
-	// TODO: remove once postgres runs in CI
-	if os.Getenv("CI") == "true" {
-		t.SkipNow()
-	}
 	config := newPachdConfig()
-	require.NoError(t, os.Setenv("STORAGE_V2", "true"))
+	config.StorageV2 = true
 	require.NoError(t, testpachd.WithRealEnv(func(env *testpachd.RealEnv) error {
-		repo := "test"
+		repo := "TestListFileV2"
```

Comment:
It probably makes sense to go ahead and switch these to tests to create a unique string for the repo name.

Reply:
The repo name changes away from just `test` ended up not being necessary.  We get a new environment for every test.